fusor-ember-cli/app/controllers/rhev-options.js

```diff
@@ -62,7 +63,7 @@ export default Ember.Controller.extend(NeedsDeploymentMixin, {
             return Ember.isPresent(value) && value !== 'Default';
           }
         }),
-        AlphaNumericDashUnderscoreValidator.create({trim: false})
+        AlphaNumericDashUnderscoreValidator.create({trim: false}) && LengthValidator.create({max: 40})
```

Comment:
LengthValidator should be passed to the All validator as another array argument, not an && condition (this will always just return the alphanumeric).